R/g.part2.R

```diff
@@ -141,6 +141,7 @@ g.part2 = function(datadir = c(), metadatadir = c(), f0 = c(), f1 = c(),
           }
         }
         IMP = g.impute(M, I,
+                       acc.metric = params_general[["acc.metric"]],
```

Comment:
It is usually safer to add new arguments to the end of the argument list, because the function may have been used without explicit calls to function arguments, which would cause errors.

For example if we had myfun(a = 1, b = "A", c = 4) then some may have used it as myfun(1, "A", 4) as that always works. However, if we then suddenly add myfun(a = 1, d = TRUE, b = "A", c = 4) then myfun(1, "A", 4) will no longer work.